src/UICrafter.Client/Components/AppViewProperties.razor

```diff
@@ -1,38 +1,45 @@
-<MudStack Row="true">
-    <MudOverlay Visible="Visible" DarkBackground="true" Absolute="true">
-        <MudProgressCircular Color="Color.Primary" Indeterminate="true" />
-    </MudOverlay>
-
-    <MudTextField @bind-Value="@AppView.Name" Label="App-view name" Variant="Variant.Outlined"></MudTextField>
-
-    <MudButtonGroup Variant="Variant.Outlined" Class="uicrafter-tooltip-buttongroup ma-2">
-        <MudTooltip Text="Public" Arrow="true" Placement="Placement.Bottom" Class="uicrafter-tooltip-buttongroup">
-            <MudToggleIconButton @bind-Toggled="AppView.IsPublic" Variant="Variant.Outlined" Icon="@Icons.Material.Filled.PublicOff" ToggledIcon="@Icons.Material.Filled.Public" />
-        </MudTooltip>
-        <MudTooltip Text="Save" Arrow="true" Placement="Placement.Bottom">
-            <MudIconButton Icon="@Icons.Material.Filled.Save" OnClick="@SaveAppView" />
-        </MudTooltip>
-        <MudTooltip Text="Delete" Arrow="true" Placement="Placement.Bottom">
-            <MudIconButton Icon="@Icons.Material.Filled.Delete" OnClick="@DeleteAppView" />
-        </MudTooltip>
-        <MudTooltip Text="Clear" Arrow="true" Placement="Placement.Bottom">
-            <MudIconButton Icon="@Icons.Material.Filled.Clear" OnClick="@ClearAppView" />
-        </MudTooltip>
-        <MudTooltip Text="Load" Arrow="true" Placement="Placement.Bottom">
-            <MudIconButton Icon="@Icons.Material.Filled.Download" OnClick="@LoadAppView" />
-        </MudTooltip>
-    </MudButtonGroup>
+<MudStack>
+    @if (AppView is not null)
+    {
+        <MudOverlay Visible="Visible" DarkBackground="true" Absolute="true">
+            <MudProgressCircular Color="Color.Primary" Indeterminate="true" />
+        </MudOverlay>
+
+        <MudTextField @bind-Value="@AppView.Name" Label="App-view name" Variant="Variant.Outlined"></MudTextField>
+
+        @if (AppView.CreatedAt != DateTime.MinValue && AppView.UpdatedAt != DateTime.MinValue)
+        {
+            <MudDatePicker Date="@AppView.CreatedAt" Label="Created at" Variant="Variant.Outlined" ReadOnly="true" />
```

Comment:
If you need to add ReadOnly="true" you can just do ReadOnly - This is also true if you want to add Row="true" you can just do Row.